frontend/src/scenes/feature-flags/featureFlagLogic.tsx

```diff
@@ -290,6 +293,26 @@ export const featureFlagLogic = kea<featureFlagLogicType>({
                 actions.setMultivariateOptions(null)
             }
         },
+        saveNewTag: ({ tag }) => {
```

Comment:
think `featureFlagTags` should listen on this as well? Add this tag to the list when created